src/argoRoleChecks.ts

```diff
@@ -57,6 +57,15 @@ export const isRdpcMember = (permissions: string[]): boolean => {
   }
 };
 
+/**
+ * check if a given set of permissions has rdpc write access
+ * @param permissions
+ */
+export const isRdpcAdmin = (permissions: string[]): boolean =>
+  permissions
+    .filter(policy => policy && policy.startsWith(RDPC_PREFIX))
+    .some(code => code.includes(PERMISSIONS.WRITE));
+
```

Comment:
I think we need a way to check if you are admin of a specific RDPC. Maybe thats a separate method like `canWriteToRdpc(permissions: string[], rdpcCode: string): boolean` which can match for a specific rdpc name (like we have with programs).

Reply:
Essentially the same as this: https://github.com/icgc-argo/ego-token-utils/pull/75#discussion_r1279875531

Reply:
OK I've added a `canWriteToRdpc` util: https://github.com/icgc-argo/ego-token-utils/pull/76/commits/1cf5f73d79dca9e6ea4d2ed11b1b4f9d6c0a62ad

Looking at it I'm not sure how necessary `isRdpcAdmin` is now; that's basically saying 'can write to ANY RDPC'